spec/lib/reporting/fraud_metrics_lg99_report_spec.rb

```diff
@@ -43,6 +65,63 @@
     end
   end
 
+  describe '#user_days_to_suspension_avg' do
+    context 'when there are suspended users' do
+      it 'returns average time to suspension' do
+        expect(report.user_days_to_suspension_avg).to eq(1.5)
+      end
+    end
+
+    context 'when there are no users' do
+      before do
+        user6.destroy
+        user7.destroy
+      end
```

Comment:
would be nice if there was a skip_let [list] macro.

Reply:
you could override by doing
```ruby
let(:user6) { nil }
let(:user7} { nil }
```
and the users wouldn't get created

Reply:
Yeah that works! I didn't think that would happen in that order.

Reply:
@zachmargolis I had to add a changelog commit so I made the `let` changes.